HelloWorld/HelloWorld/Program.cs

```diff
@@ -18,6 +18,9 @@ public double Calculate(double radis)
         return Pi * radis * radis;
     }
 
+    public double  calculaterect1(double lenth, double width)
```

Comment:
Why are you putting a 1 on the function name?

You should also use title case to name a function.
IE: public double CalculateValue(double incomingValue).

Please refer to this link for information on standard naming practices.
https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions

Reply:
What would you suggest I name this function to maintain clarity?

Reply:
```suggestion
    public double  calculaterect1(double lenth, double width)
```
public double  CalculateAreaOfRectangle(double length, double width)